maintainers/maintainer-list.nix

```diff
@@ -13000,5 +13000,11 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
+  giang = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
@SuperSandro2000 I've already put it in a separated commit, or do you mean another PR as well?

Reply:
@SuperSandro2000 https://github.com/NixOS/nixpkgs/pull/146663